pkgs/development/compilers/cppfront/default.nix

```
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+}:
+
+stdenv.mkDerivation {
+  pname = "cppfront";
+  version = "unstable-2023-05-15";
+
+  src = fetchFromGitHub {
+    repo = "cppfront";
+    owner = "hsutter";
+    rev = "ebe5ebc11303b11d29d6d40d498372c2bea07abd";
+    sha256 = "sha256-PtRmN2HsDC6byXWR2iFepvEg1KBNeIsYm/LKqlFlZ7Y=";
```

since the hash is describing it salve you should use hash instead of sha256
```suggestion 
    hash = "sha256-PtRmN2HsDC6byXWR2iFepvEg1KBNeIsYm/LKqlFlZ7Y=";
```

---

I find the ergonomics of using sha256 better than hash.